core/vm.c

```diff
@@ -75,6 +75,7 @@ int hax_vm_set_qemuversion(struct vm_t *vm, struct hax_qemu_version *ver)
             vm->features |= VM_FEATURES_FASTMMIO_EXTRA;
         }
     }
+    vm->qemu_api_version = ver->cur_version;
```

Comment:
Since vm->qemu_api_version == ver->cur_version, and it is almost impossible that different VMs use different QEMU API version, it does not seem necessary to save qemu_api_version inside vm_t.

Reply:
ver->cur_version is a parameter> to use it later I save it. But when moving to flags checking it is not needed.